Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocDateTimeValue.css

```diff
@@ -1,5 +1,17 @@
 /* Style applied to the BocDateTimeValue. */
 
+.bocDateTimeValue,
+.bocDateValue
+{
+  min-height: var(--remotion-themed-min-height);
+}
+
+.bocDateTimeValue input,
+.bocDateValue input
+{
+  box-sizing: border-box;
+}
+
 span.bocDateTimeValue,
 span.bocDateValue
 {
```

Comment:
height, see BocTextValue. 

drop overflow? (I think we dropped it in BocTextValue?)